frontend/src/index.jade

```diff
@@ -32,10 +32,10 @@ html
           .tab-content
             .tab-panes
               #tab-authorship.tab-pane(v-if="isTabAuthorship")
-                .toolbar
-                  a(onclick="expandAll(true)") Expand all
-                  span &nbsp;/&nbsp;
-                  a(onclick="expandAll(false)") Collapse all
+                .toolbar(v-if="isCollapsed && tabInfo.tabAuthorship.totalCommits > 0")
+                  a(onclick="expandAll(true)", v-on:click="isCollapsed=false") Expand all
+                .toolbar(v-if="!isCollapsed && tabInfo.tabAuthorship.totalCommits > 0")
+                  a(onclick="expandAll(false)", v-on:click="isCollapsed=true") Collapse all
```

Comment:
```
.toolbar(v-if="tabInfo.tabAuthorship.totalCommits > 0")
    a(v-on:click="isCollapsed=false", onclick="expandAll(true)") Expand all
    a(v-on:click="isCollapsed=true", onclick="expandAll(false)") Collapse all
```

Reply:
@ongspxm If ```isCollapsed``` is defined in ```v_authorship.js``` instead of ```main.js```, how do i retrieve that information in toolbar section?